android/local.properties

```diff
@@ -0,0 +1,9 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Fri Mar 15 18:40:56 CDT 2019
+ndk.dir=/home/chris/Android/Sdk/ndk-bundle
```

Comment:
Is there a way to automate this path?

Reply:
`$ANDROID_HOME` in this context points to `/home/chris/Android/Sdk`

so `$ANDROID_HOME/ndk-bundle`

Reply:
added 18869398